bom-internal/pom.xml

```diff
@@ -62,7 +62,7 @@ POM as their parent.
         <hoverfly-version>0.14.0</hoverfly-version>
         <logback.version>1.2.11</logback.version>
         <aws.version>2.17.186</aws.version>
-        <powermock.version>2.0.4</powermock.version>
```

Comment:
When I bumped our localstack-utils version to 0.2.22, the RequireUpperBoundDeps enforcer rule failed because localstack-utils pulls in powermock version 2.0.9 so I bumped the version in our pom to 2.0.9. I don't think localstack-utils will be affected when we remove powermock in the future